plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -176,6 +175,7 @@
 import static io.trino.plugin.iceberg.IcebergTableProperties.FILE_FORMAT_PROPERTY;
 import static io.trino.plugin.iceberg.IcebergTableProperties.FORMAT_VERSION_PROPERTY;
 import static io.trino.plugin.iceberg.IcebergTableProperties.PARTITIONING_PROPERTY;
+import static io.trino.plugin.iceberg.IcebergTableProperties.SCHEMA_NAME_MAPPING;
```

Comment:
Could you add a bit of rationale to this commit about how it's required for page indexing feature ?

Reply:
Native Parquet writer doesn't support writing column index, so we need to use legacy writer. However, the writer doesn't write field-id and the connector can't read the fields correctly. 

I think it would be better to suspend this PR until native Parquet writer improvement.